app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    phrase = phrase.lower()
+    letter = letter.lower()
+    for char in phrase:
+        if char == letter:
+            count += 1
+    return count
```

Comment:
The solution is correct, but it's better to use the `count()` function as it will simplify the code.

Reply:
@NazarSheremet added new commit to the branch. Please, review the updated solution